src/main/java/ca/uhn/fhir/jpa/starter/service/HelperService.java

```diff
@@ -345,15 +421,15 @@ public ResponseEntity<LinkedHashMap<String, Object>> createUsers(@RequestParam("
 				practitioners.add(practitioner.getTelecom().get(0).getValue());
 				PractitionerRole practitionerRole = FhirResourceTemplateHelper.practitionerRole(role, qualification, practitioner.getIdElement().getIdPart(), organizationId);
 //				practitionerRoleId = createResource(practitionerRole, PractitionerRole.class, PractitionerRole.PRACTITIONER.hasId(practitionerId));
-				UserRepresentation user = KeycloakTemplateHelper.user(firstName, lastName, email, keycloakUserName, initialPassword, phoneNumber, countryCode, practitioner.getIdElement().getIdPart(), practitionerRole.getIdElement().getIdPart(), role, state, lga, ward, facilityUID, argusoftIdentifier);
+				UserRepresentation user = KeycloakTemplateHelper.user(firstName, lastName, email, keycloakUserName, initialPassword, phoneNumber, countryCode, practitioner.getIdElement().getIdPart(), practitionerRole.getIdElement().getIdPart(), role, state, lga, ward, facilityUID, argusoftIdentifier, countryName);
 				String keycloakUserId = createKeycloakUser(user);
 				if (keycloakUserId == null) {
 					map.put("User not created", s);
```

Comment:
Also add any exception stack trace message caught in createKeycloakUser function